src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -253,6 +253,7 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
           Failure(regrets)
         case success => success
       }
+      case _ => //nothing
```

Comment:
perhaps doing recs.collect instead of recs.map would be better, then there is no need for the strange case _

but either way this is strange because of the non-functional (i.e. side effects) nature of this code.